Views/Home.cshtml

```diff
@@ -35,11 +35,11 @@
                     @Model.BodyText
                 }
                 <div class="[ button-holder -margin-top ]">
-                    <a class="[ button -icon-left ]" href="https://twitter.com/intent/tweet?hashtags=H5YR">
-                        <svg class="[ icon icon--button-twitter ]">
-                            <use xlink:href="#sprite-icon-twitter"></use>
+                    <a class="[ button -icon-left ]" href="https://umbracocommunity.social/share?text=%23H5YR+">
```

Comment:
The only issue with this is some folks might not be on umbracocommunity.social

So I was trying to think of a way of configuring that

Reply:
There is some really interesting discussions that have been going on for a couple of years around this sort of thing:

https://github.com/mastodon/mastodon/issues/2291
https://github.com/mastodon/mastodon/issues/19679#issuecomment-1301180085

From a quick skim of them, currently there doesn't appear to be an agreed upon solution for this because of a lack of support for Browsers/Devices.
It seems like using protocol handlers is likely going to be the final solution in some sort of guise just don't know what that will look like yet.

Reply:
Some more links to help :

https://github.com/activitypub-schema/proposal
https://codeberg.org/fediverse/fediverse-ideas/issues/1

Reply:
With protocol handlers, there always used to be a way to fallback to a URL using a timeout. You could then display a popup asking which server you belong to (perhaps defaulting to `umbracocommunity.social`) like a 3rd party mastodon server does for "Follow", "Like" and "Boost" when not logged in.

![Screenshot of the dialog described above](https://github.com/H5YR/High5YouRock/assets/3120611/7d269666-ab4b-4b3c-91e2-84d23350ee1a)

Reply:
Looking at this.

Reply:
@glombek take a look at this post, prompts user to fill out their server
https://www.bentasker.co.uk/posts/documentation/general/adding-a-share-on-mastodon-button-to-a-website.html

Obviously could make it prettier with a custom dialog UI, than a plain ole JS prompt